bahmni-commons-omod/src/test/java/org/bahmni/module/bahmnicommons/web/v1_0/controller/search/BahmniAppointmentsPatientSearchControllerTest.java

```diff
@@ -0,0 +1,76 @@
+package org.bahmni.module.bahmnicommons.web.v1_0.controller.search;
+
+import org.bahmni.module.bahmnicommons.BaseIntegrationTest;
+import org.bahmni.module.bahmnicommons.contract.patient.response.PatientResponse;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.Matchers;
+import org.mockito.Mock;
+import org.openmrs.*;
+import org.openmrs.api.PatientService;
+import org.openmrs.module.webservices.rest.web.resource.impl.AlreadyPaged;
+import org.openmrs.module.webservices.rest.web.response.ResponseException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+public class BahmniAppointmentsPatientSearchControllerTest extends BaseIntegrationTest {
+    
+    @Mock
+    private HttpServletRequest httpServletRequest;
+
+    @Mock
+    private HttpServletResponse httpServletResponse;
+
+    @Mock
+    private PatientService patientService;
+    
+    @Autowired
+    private BahmniAppointmentsPatientSearchController bahmniAppointmentsPatientSearchController;
```

Comment:
It's ok to name this `ctrl` or `controller`. The context of the test class is enough.

Reply:
Agreed! I've renamed it.